Add the ability to get the values safely

rcasula opened the pull request:
This PR is really useful in cases like PIKO 3.0. At the moment if you try to get string2 value in a PIKO 3.0 you get a index out of bound exception. 

I will make a PR to the kostalpiko-sensor-homeassistant repo to use these functions instead of raw_value[index].

Commit:
get values safely

Commit:
rebranding

Commit:
wip

gieljnssns commented on setup.py:
```diff
@@ -1,13 +1,13 @@
-    name="kostalpyko",
-    version="v0.5",
-    packages=["tests", "kostalpyko"],
+    name="kostalpiko",
```
Why did you rename this?

gieljnssns commented:
@rcasula 
Don't understand me wrong. 
I like your new code, but I don't know why you gave this package a new name.

Commit:
wip

Commit:
Fix indices

Commit:
update version

Commit:
Update changelog

Commit:
Add fixtures for tests

Commit:
Safely get values

Commit:
Add test coverage for PIKO 5.5

Commit:
Delete old tests

Commit:
Update version

Commit:
Changelog

rcasula committed:
Merge pull request #1 from rcasula/bugfix/fix-indices-and-safely-get-values

Fix indices and safely get values

maubreville committed:
Added option for 2 strings with 3 phases

In the PV inverter here I found an installation with three phases and 2 strings, which was not covered up until now for this library.

maubreville committed:
Update kostalpiko.py

Added option for 2 strings / 3 phases, which is apparently also possible :-) 

Thanks for this great library!

maubreville committed:
Update / fix for last commit

Forgot to import the new constant DOUBLE_STRING_THREE_PHASES_INDICES

maubreville committed:
Update const.py

Shouldn't have done a PR close to midnight ;-) Thanks for catching the bug with the indices.

rcasula committed:
Merge pull request #2 from maubreville/master

PR for configuration with 2 strings and 3 phases

Abildgren committed:
Update to 0.5

Make latest PR (3 phases 2 strings) available in Pypi

petos commented:
Can we get, please, merged this (and eventually revert the rename only)? I'm waiting for this merge and as far as I found there are more ppl with this issue,...

rcasula committed:
Merge pull request #3 from Abildgren/patch-2

Update to 0.5

Commit:
Update changelog

Commit:
Fix some things

Commit:
Add tests for 2 strings single phase

Commit:
remove useless

Commit:
add double strings single phase

Commit:
Bump version
